twitwi/normalizers.py

```diff
@@ -626,8 +632,8 @@ def normalize_tweet_v2(tweet, *, users_by_screen_name, places_by_id, tweets_by_i
         'text': unescape(text),
         'url': format_tweet_url(user['username'], tweet['id']),
         'hashtags': sorted(hashtags),
-        'mentioned_names': sorted(mentions),
-        'mentioned_ids': sorted(users_by_screen_name[name]['id'] for name in mentions),
```

Comment:
sorted should be on mentions and not on the list comprehension's result, in order to keep the same ordering between ids and names. This should be fixed on master